src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -652,6 +652,21 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
     </div>
   );
 
+  const renderShowShoutouts = () => (
+    <div className="toggle-room">
+      <h4 className="italic input-header">Show shoutouts</h4>
+      <label id="showShoutouts" className="switch">
+        <input
+          type="checkbox"
+          id="showShoutouts"
+          name="showShoutouts"
+          ref={register}
+        />
+        <span className="slider round"></span>
+      </label>
+    </div>
+  );
```

Comment:
The current 'default' is to show reactions. Therefore in implementing this feature, we need to ensure that current functionality is preserved. Therefore, we need to ensure that this defaults to showing reactions for everywhere it's not defined.

This same need/pattern was followed in the the PR I provided as reference (https://github.com/sparkletown/sparkle/pull/1657) using `DEFAULT_SHOW_SCHEDULE`, and should be done here as well.